### DISCUSSION TITLE
Compatibilidad con IPTV multicast

### DISCUSSION THREAD

**Discussion by CarlosRodriguez1982** [Paco8/plugin.video.movistarplus#35] [General]
Buenos días.

Ya se que Movistar encriptó su iptv multicast. Mi pregunta es si no existe una api abierta para desencriptar para los usuarios de pago. La verdad no se si habria alguna manera legal de hacerlo. Lo digo mas que nada porque la calidad de su multicast siempre es mejor. Aunque te reconozco que lo util con la tdt por internet es verlo en diferido y poder avanzar para lo que te interese.

Un saludo.

**Comment by CarlosRodriguez1982** [comment]
Mas que nada lo digo por la gente que ve el futbol en directo, porque parece que el 4K del futbol con el deco (multicast) comparado con el 4K para dispositivos, la diferencia es enorme.

**Comment by Paco8** [comment]
En Orange también hay varios canales IP codificados (los de fútbol), así que estaría bien saberlo. Yo no tengo ni idea, quizás el deco lleve la clave para descodificarlos.
